src/logic/actions/ViewPortActions.ts

```diff
@@ -113,13 +113,14 @@ export class ViewPortActions {
     }
 
     public static translateViewPortPosition(direction: Direction) {
-        if (EditorModel.viewPortActionsDisabled) return;
+        if (EditorModel.viewPortActionsDisabled || GeneralSelector.getZoom() === ViewPointSettings.MIN_ZOOM) return;
 
         const directionVector: IPoint = DirectionUtil.convertDirectionToVector(direction);
         const translationVector: IPoint = PointUtil.multiply(directionVector, ViewPointSettings.TRANSLATION_STEP_PX);
         const currentScrollPosition = ViewPortActions.getAbsoluteScrollPosition();
         const nextScrollPosition = PointUtil.add(currentScrollPosition, translationVector);
         ViewPortActions.setScrollPosition(nextScrollPosition);
+        EditorModel.mousePositionOnViewPortContent = PointUtil.add(EditorModel.mousePositionOnViewPortContent, translationVector);
```

Comment:
to jest statyczne pole w tej klasie EditorModel? 
ja bym unikał takich przypisywań - raczej wziąłbym to w jakąś metodę `EditorModel.set...` albo nawet `.update...` i wystarczy że podasz jako argument `translationVector`